src/quo/components/wallet/account_permissions/view.cljs

```diff
@@ -24,7 +24,7 @@
         [:address [:maybe :string]]
         [:emoji [:maybe :string]]
         [:customization-color [:maybe [:or :string :keyword]]]]]
-      [:token-details {:optional true} [:maybe [:vector required-tokens/?schema]]]
```

Comment:
Note: `:vector` is rarely the schema we want to use because it will fail when lazy sequences are passed.